src/design-tokens.yaml

```diff
@@ -34,6 +34,16 @@ color:
     "50": &color-teal-50 "#00b795"
     "70": &color-teal-70 "#00866d"
 
+size:
+  pixels:
+    "768": &size-pixels-768 "768px"
```

Comment:
These names seem a little weird to me since the numeric value is in the name.  Could we use the same `sm-min`, `sm-max`, etc. naming that we use in the decisions section?

Reply:
Doh! I meant to change those names. Thanks @Blackbaud-PaulCrowder